samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -1187,6 +1187,16 @@ mfxStatus ParseAdditionalParams(msdk_char *argv[], mfxU32 argc, mfxU32& i, Trans
     {
         InputParams.nVuiNalHrdParameters = MFX_CODINGOPTION_OFF;
     }
+#if (defined(_WIN32) || defined(_WIN64)) && (MFX_VERSION >= 1031)
```

Comment:
why did you move the block? you distant windows/linux adapter options from each other...

Reply:
On windows we hit compiler limit here. Too long if/else chain lead to compiler error. This function specifically designed to workaround this limitation.